### PR TITLE
add padding to .entry-title and .entry-meta on mobile

### PR DESCRIPTION
**Pull request by purzlbaum**

### CODE REVIEW

File: css/blocks.css [hunk 1]
```diff
@@ -111,7 +113,9 @@ ul.wp-block-latest-posts.is-grid.alignwide {
   .wp-block-cover-text p {
     padding: 1.5em 0;
   }
-
+
```
Comment (ntwb) [inline css/blocks.css]: Please remove the whitespace added on this line

File: css/blocks.css [hunk 2]
```diff
@@ -149,4 +153,4 @@ ul.wp-block-latest-posts.is-grid.alignwide {
   left: 0;
   bottom: 0;
   right: 0;
-}
\ No newline at end of file
+}
```
Comment (ntwb) [inline css/blocks.css:156]: Please add back the blank end-of-file line

**Review by ntwb — CHANGES_REQUESTED** [review]
Thanks @purzlbaum, please see review comments

**Comment by purzlbaum** [comment]
@ntwb I removed the blank on end-of-line: https://github.com/WordPress/gutenberg-starter-theme/pull/37/commits

**Review by ntwb — APPROVED** [review]

**Comment by karmatosed** [comment]
Can you please review the new CSS changes and send a new PR in.

**Comment by karmatosed** [comment]
@purzlbaum looking at this there are conflicts with blocks.css can you address these please?

**Comment by purzlbaum** [comment]
@karmatosed looks like someone came up with the .entry-content * solution. I've closed my PR.

**Pull request closed without merging**